.docker/Dockerfile-build

```diff
@@ -28,9 +28,9 @@ COPY --from=builder /usr/bin/kratos /usr/bin/kratos
 
 # By creating the sqlite folder as the ory user, the mounted volume will be owned by ory:ory, which
 # is required for read/write of SQLite.
-RUN mkdir -p /var/lib/sqlite
-RUN chown ory:ory /var/lib/sqlite
-VOLUME /var/lib/sqlite
+#RUN mkdir -p /var/lib/sqlite
+#RUN chown ory:ory /var/lib/sqlite
+#VOLUME /var/lib/sqlite
 
```

Comment:
Please ignore the changes to this file for now. I'll update it to the original values, when all required changes are implemented. We had to update this file to be able to build kratos locally. Please don't ask why, but it didn't work without these changes on our side.